app/assets/stylesheets/application.css.scss

```diff
@@ -6,3 +6,16 @@
 @import "daplog/base";
 @import "daplog/blocks";
 @import "daplog/contexts";
+
+#foo{}
```

Comment:
Empty rule<br>Avoid using id selectors<br>Opening curly brace in a single line rule set `{` should be preceded by at least one space